src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+
+  const convertedObj = {};
```

Comment:
[CODE STYLE] - The variable name `convertedObj` is not descriptive enough. Try to use more meaningful names that describe the purpose of the variable.